modulefiles/tasks/cheyenne/miniconda_regional_workflow.lua

```diff
@@ -1,4 +1,5 @@
 unload("python")
+load("conda")
```

Comment:
@padhrigmccarthy, is this necessary? I am asking this because this change will affect the FV3-LAM runs. I'd like to hear from other code managers before I approve this change.

Reply:
I don't think this should be added.  The necessary `regional_workflow` conda environment is loaded below this point.  No python or conda should be loaded until `miniconda3/4.12.0` has been loaded.

@padhrigmccarthy Is there a reason that `load("conda")` was added before the load of `miniconda3/4.12.0`?